gotext.go

```diff
@@ -132,38 +158,63 @@ func GetLibrary() string {
 	return lib
 }
 
-// SetLibrary sets the root path for the loale directories and files to be used at package level.
+// SetLibrary sets the root path for the locale directories and files to be used at package level.
 // It reloads the corresponding Translation file.
 func SetLibrary(lib string) {
 	globalConfig.Lock()
 	globalConfig.library = lib
 	globalConfig.Unlock()
 
-	loadStorage(true)
+	loadLocales(true)
 }
 
-// GetStorage is the locale storage getter for the package configuration.
-func GetStorage() *Locale {
+func GetLocales() []*Locale {
 	globalConfig.RLock()
-	storage := globalConfig.storage
-	globalConfig.RUnlock()
+	defer globalConfig.RUnlock()
+	return globalConfig.locales
+}
 
-	return storage
+// GetLocale is the locale storage getter for the package configuration.
+//
+// It returns the first locale returned by GetLocales.
+// This function exists for backwards compatibility, prefer using GetLocales directly.
+func GetLocale() *Locale {
+	locales := GetLocales()
+	if len(locales) > 0 {
+		return GetLocales()[0]
```

Comment:
Shouldn't we `return locales[0]` here to avoid the double function call?

Reply:
Of course, this was my oversight. Fixed.